.github/workflows/build-and-release.yml

```diff
@@ -0,0 +1,156 @@
+---
+name: Build and Release
+on: workflow_dispatch
+jobs:
+  build:
+    name: Build and Commit
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout the specific branch/ref
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+
+      - name: Fetch and checkout built branch
+        run: |
+          if git ls-remote --exit-code origin built; then
+            git fetch origin built
+            git checkout built
+            git pull origin built
+          else
+            git checkout -b built develop
```

Comment:
@acicovic do you think we should checkout the branch from `trunk` instead of `develop`? I think it might make more sense to do it, considering the release steps.

Reply:
You mean merging to trunk instead of develop?

Reply:
This checkout command creates a `built` branch from `develop` if it doesn't exist (should only happen the first time we run the Action). However, my question is if it should be `trunk` instead, and I feel like the answer should be 'yes'.

Reply:
Yes, should be trunk, after any changes are merged so we can catch them.